src/main/java/com/cavalcantgus/book_recommendation_system/controllers/GenreController.java

```diff
@@ -0,0 +1,87 @@
+package com.cavalcantgus.book_recommendation_system.controllers;
+
+import java.net.URI;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import com.cavalcantgus.book_recommendation_system.entities.Genre;
+import com.cavalcantgus.book_recommendation_system.exceptions.ResourceNotFoundException;
+import com.cavalcantgus.book_recommendation_system.services.GenreService;
+import com.cavalcantgus.book_recommendation_system.util.PatcherUpdater;
+
+@RestController
+@RequestMapping("/genres")
+public class GenreController {
+
+	private static final Logger logger = LoggerFactory.getLogger(GenreController.class);
+
+	@Autowired
+	private GenreService service;
+
+	@Autowired
+	private PatcherUpdater patcher;
+
+	@GetMapping
+	public ResponseEntity<List<Genre>> findAll() {
+		List<Genre> list = service.findAll();
+		return ResponseEntity.ok().body(list);
```

Comment:
se não vai tratar o retorno, pode simplesmente retornar direto. fica mais conciso e fácil de ler

```suggestion
		return ResponseEntity.ok().body(service.findAll());
```